docs/interpreter/cassandra.md

```diff
@@ -793,7 +812,17 @@ Below are the configuration parameters and their default value.
 
 <hr/>
 
-## 14. Bugs & Contacts
+## 14. Change Log
+
+**2.0** : 
```

Comment:
this versioning might be a bit confusing? I don't know if we have separate version for built-in interpreters but it is definitely useful to highlight changes.